pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -64,15 +64,18 @@ type MigPlanSpec struct {
 	MigStorageRef     *kapi.ObjectReference `json:"migStorageRef,omitempty"`
 	Closed            bool                  `json:"closed,omitempty"`
 	Hooks             []MigPlanHook         `json:"hooks,omitempty"`
+	Refresh           bool                  `json:"refresh,omitempty"`
 }
 
 // MigPlanStatus defines the observed state of MigPlan
 type MigPlanStatus struct {
 	UnhealthyResources
 	Conditions
-	Incompatible      `json:",inline"`
-	ObservedDigest    string   `json:"observedDigest,omitempty"`
-	ExcludedResources []string `json:"excludedResources,omitempty"`
+	Incompatible       `json:",inline"`
+	ObservedDigest     string         `json:"observedDigest,omitempty"`
+	ExcludedResources  []string       `json:"excludedResources,omitempty"`
+	SrcStorageClasses  []StorageClass `json:"srcStorageClasses,omitempty"`
```

Comment:
There is enough code in mig-ui that relies on data meeting the interface defined by `StorageClass[]` and pulled off a kube resource that I think it's too much work to convert us over to using the discovery service for StorageClasses at this time. 

I will make a separate PR adding StorageClasses to discovery service so that this can be fixed with less effort in the future if prioritized.